.gitignore

```diff
@@ -0,0 +1,8 @@
+# Secrets
+.streamlit/secrets.toml
+
+# environment
+briefgpt/*
```

Comment:
good practice for you, but we don't really want to add our venvs into the .gitignore, otherwise that could potentially be a lot of collaborators venvs :)